src/js/index.js

```diff
@@ -47,7 +47,7 @@ async function init (flags) {
 	$$('.video__actions__download').forEach(el => el.parentNode.removeChild(el));
 }
 
-export {
-	init,
-	checkIfUserIsSyndicationCustomer
-};
+// export {
+// 	init,
+// 	checkIfUserIsSyndicationCustomer
+// };
```

Comment:
No commented out code please

Reply:
Fair point missed that in my clean up

Reply:
There is some more as part of a  //TODO in message.js that is there for discussion. I'm not 100% on the whole 'rich_article' status of a user.  See notes on that in messages.js